src/ImageSharp/Formats/Gif/GifEncoderCore.cs

```diff
@@ -132,17 +134,19 @@ public void Encode<TPixel>(Image<TPixel> image, Stream stream)
         /// <returns>
         /// The <see cref="int"/>.
         /// </returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
I know, I did it several times, but it's most likely not worth to inline methods with complex loops inside them.